ui/src/literature/containers/CitationSummaryBoxContainer.jsx

```diff
@@ -2,7 +2,7 @@ import { connect } from 'react-redux';
 
 import { LITERATURE_NS } from '../../reducers/search';
 import CitationSummaryBox from '../components/CitationSummaryBox';
-import { convertAllImmutablePropsToJS } from '../../common/immutableToJS';
+// import { convertAllImmutablePropsToJS } from '../../common/immutableToJS';
```

Comment:
even if it's hot fix should we keep it like that?

Reply:
ah left over